jsonpb/jsonpb_test.go

```diff
@@ -684,7 +731,35 @@ func (m *dynamicMessage) MarshalJSONPB(jm *Marshaler) ([]byte, error) {
 	return []byte(m.rawJson), nil
 }
 
-func (m *dynamicMessage) UnmarshalJSONPB(jum *Unmarshaler, json []byte) error {
-	m.rawJson = string(json)
+func (m *dynamicMessage) UnmarshalJSONPB(jum *Unmarshaler, js []byte) error {
+	m.rawJson = string(js)
 	return nil
 }
+
+func (m *dynamicMessage) Marshal() ([]byte, error) {
```

Comment:
You're right. If I add a properly formatted struct tag to the field, this is no longer necessary. Done.